go.mod

```diff
@@ -24,40 +25,44 @@ require (
 	github.com/mattn/go-colorable v0.1.13
 	github.com/mattn/go-isatty v0.0.16
 	github.com/mccutchen/go-httpbin v1.1.2-0.20190116014521-c5cb2f4802fa
-	github.com/mstoykov/atlas v0.0.0-20220808085829-90340e9998bd
+	github.com/mstoykov/atlas v0.0.0-20220811071828-388f114305dd
```

Comment:
hmm :thinking: @mstoykov, I think we forgot to update this in k6 after some optimizations :sweat_smile:

Reply:
I guess so - the optimization likely matters very little currently in k6.

If we want we can update it outside of this PR?